traffic_ops/traffic_ops_golang/cdn/queue.go

```diff
@@ -56,22 +75,76 @@ func Queue(w http.ResponseWriter, r *http.Request) {
 		api.HandleErr(w, r, inf.Tx.Tx, http.StatusNotFound, nil, nil)
 		return
 	}
+
```

Comment:
I think we also need to add the `cdnID` entry in the `queryParams` map. As I type that, I realize `queryParams` probably isn't the best name since `cdnID` would actually be coming from a _path_ param. Maybe just call it `params`?

Reply:
For some reason, I had a mix up with the commit. Should be fixed now.